Sources/FaunaDB/Value.swift

```diff
@@ -1,7 +1,58 @@
 import Foundation
 
+/**
+    `Value` is the top level of the value tree hierarchy. It represents a valid
+    FaunaDB entry returned by the server.
+
+    [Reference](https://fauna.com/documentation/queries#values)
+
+    ## Transversal API
```

Comment:
I think you mean "Traversal" instead of "Transversal", unless this is a new term that I'm unaware that means the combination of "traversal" and "transform" because that would be a sick new word